sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/_operations/_patch.py

```diff
@@ -75,7 +74,7 @@ def get_token_by_key(endpoint: str, hub: str, key: str, **kwargs: Any) -> str:
         payload["role"] = roles
     if groups:
         payload["webpubsub.group"] = groups
-    return six.ensure_str(jwt.encode(payload, key, algorithm="HS256", headers=kwargs.pop("jwt_headers", {})))
+    return jwt.encode(payload, key, algorithm="HS256", headers=kwargs.pop("jwt_headers", {})).encode("utf-8")
```

Comment:
This doesn't look the same?
The return type is `str` and this will encode to `bytes`